public/index.html

```diff
@@ -5,17 +5,39 @@
         <title>Chat</title>
         <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.1/angular.min.js"></script>
         <script src="js/main.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+        <link href="css/main.css" rel="stylesheet" type="text/css">
     </head>
     <body ng-controller="ChatController">
-        <a ng-show="!loggedIn" href="{{loginUri}}">Log in</a>
+        <button ng-show="!loggedIn" ng-click="githubLogin()">Log in with github</button><br>
+        <form ng-show="!loggedIn" ng-submit="guestLogin()">
+            <input type="text" ng-model="nameInputBox" placeholder="Username" required></input>
+            <input  type="submit" value="Log in as guest"></input>
+        </form>
         <section ng-show="loggedIn">
-            <p>Logged in as {{user.name}}.</p>
+            <p>Logged in as {{myUserData.name}}.</p>
             <h2>List of registered users</h2>
             <ul>
-                <li ng-repeat="user in users">
+                <li ng-repeat="user in registeredUsers">
                     {{user.name}}
+                    <img ng-src="{{user.avatarUrl}}" alt="{{user.name}} avatar" style="width:50px;height:50px;">
```

Comment:
Inline Style. Use CSS.
(I assume this was for testing something and you left it in)